app/src/main/java/com/lucasfagundes/androidudemycourse/feature/task_list/RecyclerItemClickListner.kt

```diff
@@ -0,0 +1,43 @@
+package com.lucasfagundes.androidudemycourse.feature.task_list
+
+import android.content.Context
+import android.view.GestureDetector
+import android.view.MotionEvent
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+
+class RecyclerItemClickListener(context: Context, recyclerView: RecyclerView, private val mListener: OnItemClickListener?) : RecyclerView.OnItemTouchListener {
```

Comment:
```suggestion
class RecyclerItemClickListener(
          context: Context, 
          recyclerView: RecyclerView, 
          private val mListener: OnItemClickListener?
          ) : RecyclerView.OnItemTouchListener {
```

o que acha de quebrar a linha dessa forma pra ficar mais legivel?